content/en/docs/contribute/intermediate.md

```diff
@@ -466,8 +471,22 @@ most up-to-date version of that branch.
     your changes applied. This is how reviewers will check your changes.
 
 9.  If you notice that more changes need to be made, or if reviewers give you
-    feedback, address the feedback locally, then repeat step 4 - 6 again,
-    creating a new commit. The new commit is added to your pull request and the
+    feedback, address the feedback locally and push another commit.
+
+    ```bash
+    git commit -a --amend
+    ```
+
+    - -a: commit all changes
+    - --amend: amend this commit to my previous commit
+
+    A editor will open so you can update your commit message if necessary.
```

Comment:
```suggestion
    An editor will open so you can update your commit message if necessary.
```

Reply:
I think you have eagle eyes. LOL.  I admit I've not been at my best this week due to a virus and will do better double-checking my own work going forward.